### ISSUE TITLE
When unbinding the inventory and crafting buttons for the keyboard, controller cannot open the inventory

### ISSUE THREAD

**Issue by relt-1** [Wilyicaro/Legacy-Minecraft#340]
When unbinding the inventory and crafting buttons for the keyboard, controller cannot open the inventory

# Reproducing
Go to the controls menu
Go to keyboard
Unbind "Crafting"
Unbind "Inventory"
Now controller will be unable to open the inventory with square or triangle
To fix it, you have to reset to defaults and restart the game

Affected version:
1.20.4
Fabric
Latest

**Comment by hahayupgit** [comment]
Can confirm, same thing happens to me. Additionally, when I went to bind the Inventory button, I set it to the `E` key, not realizing that this was the same button for the Crafting menu. When I did this, both my `X` and `Y` buttons on my Steam Deck opened the crafting menu. After changing the inventory button to something else (in this case `[`) the `Y` button successfully brought up the inventory.

Using the Steam Deck and the official modpack.
